common/common-iop.mk

```diff
@@ -43,6 +43,12 @@ IOP_LIBS += -lkernel -lgcc
 %.o : %.s
 	$(IOP_AS) $(IOP_ASFLAGS) $< -o $@
 
+%.o : %.lst
+	echo "#include \"irx_imports.h\"" > build-imports.c
```

Comment:
Humm, do you mean, have a single shared imports file for all the tests? or a single file for all the tests inside a directory?